docs/website/i18n/en.json

```diff
@@ -79,7 +79,7 @@
     "package": "Package files format",
     "Package files format": "Package files format",
     "reference": "Reference",
-    "setup": "Setup",
+    "setup": "Install and set up",
```

Comment:
Revert.

Reply:
Why do you think it's important to revert this? I think it's important for the wanna-install user to know exactly where that info is provided, since "setup" is not explicit enough.

Reply:
"Setup" already includes installation, "Installation and setup" is redundant ("Install and set up" is wrong :-\)

Reply:
I disagree, I think it's that "install/ation" is an important keyword and we have to make explicit where that step lives, specially for people looking for it.

Btw "Install and set up" are two verbs, why do you say is wrong?

Reply:
Because it's imperative, and we are using gerund or nouns.

Reply:
I'd go with installation alone if you really need it, or keep both, up to you.